src/instruments/instrumentstypes.h

```diff
@@ -171,6 +180,37 @@ struct PartInstrument {
 
 using PartInstrumentList = QList<PartInstrument>;
 
+struct ScoreOrderGroup {
+    int index;
+    QString family;
+    QString section { QString() };
```

Comment:
section will be empty by default, you don't need to call QString() explicitly

Reply:
Done.